tiflash/troubleshoot-tiflash.md

```diff
@@ -97,3 +98,167 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 1. 参照[下线 TiFlash 节点](/scale-tidb-using-tiup.md#方案二手动缩容-tiflash-节点)一节下线对应的 TiFlash 节点。
 2. 清除该 TiFlash 节点的相关数据。
 3. 重新在集群中部署 TiFlash 节点。
+
+## TiFlash 分析慢
+
+如果语句中含有 MPP 模式不支持的算子或函数等，TiDB 不会选择 MPP 模式，可能导致分析慢。此时，可以执行 `EXPLAIN` 语句检查 SQL 中是否含有 TiFlash 不支持的函数或算子。
+
+{{< copyable "sql" >}}
+
+```sql
+set @@session.tidb_enforce_mpp=1;
+create table t(a int);
+explain select count(*) from t;
+show warnings;
+```
+
+```
++---------+------+-----------------------------------------------------------------------------+
+> | Level   | Code | Message                                                                     |
++---------+------+-----------------------------------------------------------------------------+
+| Warning | 1105 | MPP mode may be blocked because there aren't tiflash replicas of table `t`. |
++---------+------+-----------------------------------------------------------------------------+
+```
+
+
```

Comment:
```suggestion
在上例中，如 warning 信息所示，因为 TiDB 5.5 及更早的版本中尚不支持 `subtime` 函数的下推，因此 TiDB 没有选择 MPP 模式。
```

Reply:
@zanmato1984 Is TiDB 5.5 right here? It seems the latest version is the to-be-released TiDB 5.4.